src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -325,7 +325,7 @@ public static void ShowMenu(View anchor, params MenuItem[] items)
         /// </summary>
         /// <param name="anchorPosition">The anchor position where menu is displayed.</param>
         /// <param name="items">The menu items.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
anchor position need to be checked again.

Reply:
As I mentioned https://github.com/Samsung/TizenFX/pull/2922#discussion_r615518743 , we can think about "Anchor" and "Anchor Position" more.

Reply:
I have added "Anchor" property and removed "AnchorPosition" property.